docs/user-guide/nexd.md

```diff
@@ -15,11 +15,12 @@ USAGE:
    nexd [global options] [command [command options]] [arguments...]
 
 COMMANDS:
-   version  Get the version of nexd
-   proxy    Run nexd as an L4 proxy instead of creating a network interface
-   router   Enable advertise-cidr function of the node agent to enable prefix forwarding.
-   relay    Enable relay and discovery support function for the node agent.
-   help, h  Shows a list of commands or help for one command
+   version    Get the version of nexd
+   proxy      Run nexd as an L4 proxy instead of creating a network interface
+   router     Enable advertise-cidr function of the node agent to enable prefix forwarding.
+   relay      Enable relay and discovery support function for the node agent.
+   relayderp  Enable DERP relay to relay traffic between nexd nodes.
```

Comment:
A single `relay` subcommand may be a little nicer.  `nexd relay derp <options>`

Reply:
Just to clarify, do you want derp as a subcommand under the relay subcommand or as an option ? I assume it's first.

The reason i kept it separate was because the wireguard relay doesn't support any of the options that derp support, so the overall cli experience might not be that great. Also if we plan to deprecate wireguard relay, it's probably going to be easy to deprecate the existing flag.

Reply:
I was thinking a subcommand, but it's not a big deal. It would primarily make sense if `relay derp` inherited all of the same config options allowed under just `relay`.